### PR TITLE
SuccessView: Make primary label format consistent

### PR DESCRIPTION
**Pull request by danrabbit**
Thinking about it, I think I like the way this looks better:

Screenshot from 2020-09-01 11 50 42@2x

### CODE REVIEW

**Review by jeremypw — APPROVED** [review]
Better

**Review by cassidyjames — COMMENTED** [review]
Ah so my comment in Slack was that I didn't love that this was a bit more passive/distanced, like, "Foo App was installed all by itself, I have no idea how it happened!" versus "I installed Foo App". The fallback text is a bit more passive just because it felt stilted to say, "Installed the app".

That said, @danrabbit and I both didn't feel very strongly either way. :stuck_out_tongue:

**Comment by cassidyjames** [comment]
Super scientific polls for an added data point:

- https://twitter.com/CassidyJames/status/1305893912561635328
- https://mastodon.social/@cassidyjames/104869692431154115

**Comment by cassidyjames** [comment]
Another phrasing someone brought up on Twitter is `“Foo App” has been installed`. It’s still distanced a bit like this, but the tense feels better? `has been` vs `was`.

File: src/Views/SuccessView.vala [hunk 1]
```diff
@@ -38,7 +38,7 @@ public class Sideload.SuccessView : AbstractView {
 
         if (view_type == SuccessType.INSTALLED) {
             if (app_name != null) {
-                primary_label.label = _("Installed “%s”").printf (app_name);
+                primary_label.label = _("“%s” was installed").printf (app_name);
```
Comment (cassidyjames) [inline src/Views/SuccessView.vala:41]:
```suggestion
                primary_label.label = _("“%s” has been installed").printf (app_name);
```

File: src/Views/SuccessView.vala [hunk 2]
```diff
@@ -38,7 +38,7 @@ public class Sideload.SuccessView : AbstractView {
...
             } else {
                 primary_label.label = _("The app was installed");
```
Comment (cassidyjames) [inline src/Views/SuccessView.vala:43]:
```suggestion
                primary_label.label = _("The app has been installed");
```

**Review by cassidyjames — COMMENTED** [review]
Suggestion from Twitter. Feels a bit better, I think, and keeps it consistent. @elementary/ux ?

**Pull request merged**